interface/themes/style_pdf.css

```diff
@@ -605,7 +605,7 @@ font-weight: bold;
     padding: 0px 0px 0px 10px;
 }
 
-.css_button {
+.css_button, a[role="button"] {
```

Comment:
I don't remember what style_pdf is for at the moment, but it is not actually a full theme...maybe this is appropriate, but I am not sure how to test at the moment.